### DISCUSSION TITLE
How to install

### DISCUSSION THREAD

**Discussion by salomaoa** [Mahmud0808/Iconify#33] [Q&A]
I would be pleased if someone coulg help me try this mod.

First problem: How to download it. I cant find a direct link to download a file (apk, zip or whatever).When I click "go to file" I just get a directory with several files...

If I ever download the file, how to instal, it? is it a .apk file? is it a .zip file? To install it I need to run magisk and install it from there? How?

Any help would be much apreciated

Salomao

**Comment by Mahmud0808** [comment]
Check [Download](https://github.com/Mahmud0808/Iconify#download) section of README. Release build is apk, debug build is zip. Extract the zip file to get apk file.

**Comment by salomaoa** [comment]
Thank you for your reply

This is what I get when I click the "release build" link.

Where do I click now to download the file?

[image: imagem.png]


DrDisagree ***@***.***> escreveu no dia terça, 10/01/2023
à(s) 06:18:

> Check Download <https://github.com/Mahmud0808/Iconify#download> section
> of README. Release build is apk, debug build is zip. Extract the zip file
> to get apk file.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Mahmud0808/Iconify/discussions/33#discussioncomment-4641570>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADXZZ7YCDEKV2DX74GWL4G3WRT5LPANCNFSM6AAAAAATWAP3RQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
______________________
Salomão Abreu

**Reply by Mahmud0808** [reply]
Release Build > Assets > Iconify.v4.0-Beta.apk
